src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -22,16 +32,35 @@ class LocalBackend extends Backend {
     commandWriter.write(commandLine)
     commandWriter.flushAndClose()
 
-    s"/bin/bash $commandFile" ! ProcessLogger(stdoutWriter write, stderrWriter write)
+    s"/bin/bash $commandFile" ! ProcessLogger(stdoutWriter writeWithNewline, stderrWriter writeWithNewline)
 
     Seq(stdoutWriter, stderrWriter).foreach {
```

Comment:
OK on Vector.  The IntelliJ Scala plugin by default really wants this to be on 3 lines.  If you update the plugin settings to force these to one line and share the updated plugin config I promise to use that.  :smile: